cypress/integration/character_management.js

```diff
@@ -0,0 +1,58 @@
+describe('Character Management Test Suite', () => {
+    const testusername = 'testuser'
+    const character_name = 'John'
+    const character_race = 'Smith'
+    const character_class = 'Upper'
+    const character_faction = 'Five'
+    const character_level = 5
+    const character_downtime = 30
+
+    before(() => {
+        // Login
+        cy.viewport(1920, 1080)
+        cy.visit('/')
+        cy.get('[data-testid="PersonIcon"]').click()
+        cy.contains('button', 'Register').click()
+        cy.get('input[id="username"]').type(testusername)
```

Comment:
for all the inputs with id, you could just do #id

Reply:
That makes more sense, thank you.